### ISSUE TITLE
The `plot_pattern()` function produces a non-informative error when complete data is supplied

### ISSUE THREAD

**Issue by hanneoberman** [amices/ggmice#44]
Problem:

``` r
ggmice::plot_pattern(na.omit(mice::boys))
#>  /\     /\
#> {  `---'  }
#> {  O   O  }
#> ==>  V <==  No need for mice. This data set is completely observed.
#>  \  \|/  /
#>   `-----'
#> Error in `stop_subscript()`:
#> ! Can't subset columns that don't exist.
#> x Columns `age`, `hgt`, `wgt`, `bmi`, `hc`, etc. don't exist.
```

<sup>Created on 2022-04-05 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Solution: add a `return()` in the preprocessing part of the function to exit the function immediately whenever the data is complete

**Comment by hanneoberman** [comment]
#76 Should fix this

**Comment by hanneoberman** [comment]
Maybe add an easter egg later?

**Comment by hanneoberman** [comment]
Fixed in #77
